index.html

```diff
@@ -47,37 +47,36 @@ <h4>LET'S CONNECT</h4>
         </div>
 
         <section id="works">
-        <div class="content service--content">
     <p>  <div class="item1">
```

Comment:
As asked by the previous reviewer, please stick as much as possible to Figma design.

Your design: 
<img width="379" alt="image" src="https://user-images.githubusercontent.com/7243132/200089659-2037b52a-ae28-4126-94ea-cade95cc26f2.png">


Figma design:
<img width="379" alt="image" src="https://user-images.githubusercontent.com/7243132/200089680-d137edea-4f74-42db-b54f-d614f1b84b03.png">

Please remove border  : 
<img width="274" alt="image" src="https://user-images.githubusercontent.com/7243132/200089920-1cbbf42f-a756-4b9f-8800-0e4e627f6010.png">

Please add padding and check border color
<img width="274" alt="image" src="https://user-images.githubusercontent.com/7243132/200089940-c230fee1-c7bf-449e-a83f-f9b315b01d94.png">

Please check space between the element and the color
<img width="274" alt="image" src="https://user-images.githubusercontent.com/7243132/200089975-df174a51-dc59-4b07-bf8b-4876406166f7.png">

Please check font-weight
<img width="274" alt="image" src="https://user-images.githubusercontent.com/7243132/200090023-95c1ac14-65a6-48e1-b4c7-13c9cdf7b5ad.png">

Please do it for each card.